resource_type.go

```diff
@@ -0,0 +1,12 @@
+package sarama
+
+type ResourceType int8
+
+const (
+	UnknownResource ResourceType = 0
```

Comment:
curious where you got this set of constants? I don't see it in the docs anywhere?

Reply:
Took me a while to track it down again. It's from [KIP-133](https://cwiki.apache.org/confluence/display/KAFKA/KIP-133%3A+Describe+and+Alter+Configs+Admin+APIs#KIP-133:DescribeandAlterConfigsAdminAPIs-WireFormattypes) 

I can add a comment in the code